CMakeLists.txt

```diff
@@ -142,9 +142,12 @@ if(GFX_ENABLE_SCENE)
     target_include_directories(gfx PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/third_party/Vulkan-Headers/include)
     target_include_directories(gfx PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/third_party/tinyexr)
     target_link_libraries(gfx PUBLIC glm::glm)
-    target_link_libraries(gfx PRIVATE tinyobjloader ktx)
+    target_link_libraries(gfx PRIVATE tinyobjloader ktx tinyexr)
 
     target_compile_definitions(gfx PUBLIC GFX_ENABLE_SCENE)
+	
+    #tinyexr cmake is currently broken when not using miniz
+    target_compile_definitions(tinyexr PRIVATE TINYEXR_USE_MINIZ=0 TINYEXR_USE_STB_ZLIB=1)
```

Comment:
Unfortunately tinyexr's cmake is broken and doesnt correctly set the required defines. So this is a workaround